README.md

```diff
@@ -90,10 +90,72 @@ To start the local webserver and have gulp watch for changes:
 npm start
 ```
 
-#### Descriptions for optional `gulp [ flags ]` task prefixes
+### Descriptions for optional `gulp [ flags ]` task prefixes
```

Comment:
This increase in heading along with the one above it for `Development` allow for better typographic headers in the `README.md`.